flixkit.go

```diff
@@ -58,6 +58,69 @@ type FlowInteractionTemplate struct {
 	Data     Data   `json:"data"`
 }
 
+type FlixService interface {
+	GetFlixRaw(templateName string) (string, error)
```

Comment:
It is best practise to add context as the first parameters to these kind of apis in order to be able to cancel a request chain, can we add that?